estuary_updater/handlers/freshmaker.py

```diff
@@ -72,3 +86,30 @@ def handle(self, msg):
                 build = ContainerKojiBuild.get_or_create(build_params)[0]
 
             event.triggered_container_builds.connect(build)
+
+    def build_state_handler(self, msg):
+        """
+        Handle a Freshmaker build state changed message and update Neo4j if necessary.
+
+        :param dict msg: a message to be processed
+        """
+        build = msg["body"]["msg"]
```

Comment:
Could you please put this code in a generic method that can be reused rather than copying the same code?